adaptive_shield/README.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+SaaS Apps like Office 365, Slack, Zoom, and Salesforce are vital to a business's day-to-day operations, but  can often introduce new security challenges. Adaptive Shield's SaaS Security Posture Management solution (SSPM) features proactive, deep, continuous and automated monitoring and management of all business critical SaaS applications. By integrating with all SaaS apps within minutes, Adaptive Shield enables security teams to gain complete control over their SaaS apps, allowing them to harden their security policies and reduce risks.
+
+With the Adaptive Shield integration, you can track and monitor SaaS posture alerts, such as configuration drifts, integration failures, and security check degradations, from within the Datadog Events Explorer.
+
+## Setup
+
+1. Click **Connect Accounts**, to begin authorization of this integration, then navigate to [Adaptive Shield][1].
```

Comment:
Clicking Connect Accounts will bring the user directly to Adaptive Shield (as long as the `onboarding_url` leads to there). 
```suggestion
1. Click **Connect Accounts** to begin authorization of this integration. You will then be redirected to [Adaptive Shield][1].
```